src/EditorFeatures/CSharpTest/Workspaces/WorkspaceTests.cs

```diff
@@ -48,7 +48,7 @@ public void TestEmptySolutionUpdate()
                 WaitForWorkspaceOperationsToComplete(workspace);
 
                 Assert.Equal(solution, workspace.CurrentSolution);
-                Assert.False(workspaceChanged);
+                Assert.True(workspaceChanged);
```

Comment:
It obviously needs to be assert true, because the whole purpose of the test is to prove the event is fired.  I don't know why it was the other way, or why the change makes the event become observable now and not before.